src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Drawable {
+    default void draw() {
+        System.out.println("Figure: " + this);
```

Comment:
let's better override this method in each child figure class - make implementations slightly different